core/modules/cart/store/actions.ts

```diff
@@ -295,7 +296,7 @@ const actions: ActionTree<CartState, RootState> = {
         continue
       }
       if (config.entities.optimize && config.entities.optimizeShoppingCart) {
-        product = omit(product, ['configurable_children', 'configurable_options', 'media_gallery', 'description', 'category', 'category_ids', 'product_links', 'stock', 'description'])
```

Comment:
This change will increase the size of the order json passed to server endpoint api/order/create. We should remove these properties in order/placeOrder action just before passing them to the server